diff --git a/apis/v1/queue/oas_client_gen.go b/apis/v1/queue/oas_client_gen.go
index 6ab5c8d..27d14d4 100644
--- a/apis/v1/queue/oas_client_gen.go
+++ b/apis/v1/queue/oas_client_gen.go
@@ -405,7 +405,8 @@ func (c *Client) sendListQueues(ctx context.Context) (res ListQueuesRes, err error
 
 	u := uri.Clone(c.requestURL(ctx))
 	var pathParts [1]string
 	pathParts[0] = "/commonserviceitem"
 	uri.AddPathParts(u, pathParts[:]...)
+	u.RawQuery = url.QueryEscape(`{"Filter":{"Provider.Class":"simplemq"}}`) // NOTE: ここだけOpenAPIで表現できず手動で書き加えている
 
 	r, err := ht.NewRequest(ctx, "GET", u)